src/views/BottomTabBar.tsx

```diff
@@ -198,12 +202,19 @@ class TabBarBottom extends React.Component<BottomTabBarProps, State> {
       return;
     }
 
-    this.setState({
-      layout: {
-        height,
-        width,
+    this.setState(
+      {
+        layout: {
+          height,
+          width,
+        },
       },
-    });
+      () => {
```

Comment:
Is it necessary to wait for the setState callback for this? Something like this should work fine:

```js
this.props.onHeightChange && this.props.onHeightChange(...)
this.setState(...)
```